### PR TITLE
Add TimeMergeImputor and flatten

### PR DESCRIPTION
**Pull request by dding3**
1. Support TimeMergeImputor to merge time which are seconds apart
2. Add flatten method to elastic_search to parse loading data
3. Fix comments

### CODE REVIEW

**Comment by dding3** [comment]
jekins passed. http://10.239.47.210:18888/job/ZOO-PR-Validation/4517/

File: pyzoo/zoo/automl/impute/impute.py [hunk 1]
```diff
@@ -42,3 +42,39 @@ class FillZeroImpute(BaseImputation):
...
+                                   * self.time_interval) - f.second(ori_column_name))\
+            .withColumn(self.timestamp_column_name,
+                        f.from_unixtime(f.unix_timestamp(ori_column_name)
+                                        + f.col("add_seconds"))).drop("add_seconds")
+        if self.mode == "max":
+            merged_df = merged_df.groupby(self.timestamp_column_name).max()
+        elif self.mode == "min":
+            merged_df = merged_df.groupby(self.timestamp_column_name).min()
+        elif self.mode == "mean":
+            merged_df = merged_df.groupby(self.timestamp_column_name).min()
```
Comment (jason-dai) [inline pyzoo/zoo/automl/impute/impute.py:72]: min?

File: pyzoo/zoo/automl/impute/impute.py [hunk 2]
```diff
@@ -42,3 +42,39 @@ class FillZeroImpute(BaseImputation):
...
+        elif self.mode == "sum":
+            merged_df = merged_df.groupby(self.timestamp_column_name).min()
```
Comment (jason-dai) [inline pyzoo/zoo/automl/impute/impute.py:74]: min?

**Comment by dding3** [comment]
jekins failed. The error is not related to this PR. 
FAILED ../test/zoo/orca/learn/jep/test_pytorch_estimator_for_spark.py::TestEstimatorForSpark::test_bigdl_pytorch_estimator_shard

Merge the PR first.

**Pull request merged**